jcl/src/java.base/share/classes/com/ibm/tools/attach/target/Attachment.java

```diff
@@ -70,11 +74,15 @@
 				try {
 					/*[IF Sidecar19-SE-B165]*/
 					agentClass = Class.forName("jdk.internal.agent.Agent"); //$NON-NLS-1$
-					startRemoteArgumentType = String.class;
 					/*[ELSE] Sidecar19-SE-B165*/
 					agentClass = Class.forName("sun.management.Agent"); //$NON-NLS-1$
-					startRemoteArgumentType = Properties.class;
 					/*[ENDIF] Sidecar19-SE-B165*/
+					
+					/*[IF Sidecar19-SE-B165 | Sidecar18-SE-OpenJ9]*/
+					startRemoteArgumentType = String.class;
+					/*[ELSE] Sidecar19-SE-B165 | Sidecar18-SE-OpenJ9
```

Comment:
It is preferred to close the comment so the following line of code is not commented out.

Reply:
Done

Reply:
Done